doc/Metrics.md

```diff
@@ -62,12 +64,65 @@ added to the deployment with the following parameters:
   "version": "1.0"
 }
 ```
-After the modules are deployed, Chronograf can be reached at localhost:8888.
+After the modules are deployed, Chronograf can be reached at http://localhost:8888. InfluxDb should be configured as a data source for Chronograf. The InfluxDb instance should be reachable at http://influxdb:8086.
 
 ## Example InfluxDb queries
 
 For example InfluxDb queries, please look at https://github.com/Azure/iotedge/pull/141/files#diff-9fc75ceaff4ae01d42e1bafad8037848
 
 ## Collecting CPU and memory stats from EdgeHub and other containers
 
-A telegraf container can be added to the deployment to collect stats about all containers in the deployment. Please refer to https://hub.docker.com/_/telegraf/ for steps to configure it.  
+A telegraf container can be added to the deployment to collect stats about all containers in the deployment. Telegraf uses a configuration file on the device to configure itself. Azure IoT Edge deployments can only mount files in to a container but cannot pass a configuration file to the device from the cloud. For this reason, the configuration has to be done on the device as follows:
```

Comment:
Technically, there is a way to do this -- albeit, somewhat roundabout. For example, we could deploy a module to Edge with the restart policy set as "never restart". And then the module has code that accepts an arbitrary string and a file path as command line arguments which we set via the JSON deployment. We volume mount a volume into the container and have it write that string into a file path in that volume. Then we have that same volume mounted in the telegraf container! :smile: